tools/ovc/openvino/__init__.py

```diff
@@ -43,12 +43,13 @@
 from openvino.runtime import Tensor
 from openvino.runtime import OVAny
 
-from openvino.runtime import compile_model
+# Helper functions for openvino module
+from openvino.runtime.utils.data_helpers import tensor_from_file
```

Comment:
Will we still have `openvino.runtime`? May be it is better to have `openvino.utils`?